### PR TITLE
Added national wonders, fixed some minor issues

### PR DESCRIPTION
**Pull request by HadeanLake**
* fixed translation of "Owned: [amountOwned] = " line
* Tweaked culture from killing
Aztecs get melee strength of the unit killed in culture and honor opener does the same thing.
They stack. So you get culture equal to 200% of the dead unit's strength (not ranged strength).
* added missing unit and building definitions from vanilla (commented out - no new units in this PR), added missing obsolete tech tags, etc
removed hurryCostModifier from units and building that should not have it
source: [civ 5 XMLs.zip](https://github.com/yairm210/Unciv/files/5252559/civ.5.XMLs.zip)
* added promotions and parametrized their uniques:
Morale (+[15]% Combat Strength)
Great Generals I ([Great General] is earned [50]% faster)
Great Generals II ([Great General] is earned [100]% faster)
Quick Study ([50]% Bonus XP gain)
* Added Heroic Epic, National Epic and National Treasury national wonders
Heroic Epic has unique "All newly-trained [non-air] units in this city receive the [Morale] promotion" - "non-air" is valid param now
93721799-f8725800-fb9a-11ea-90f8-80d48564d7d6

[Raising The Flag on Iwo Jima silhouette, Marine Corps War Memorial Raising the Flag on Iwo Jima Battle of Iwo Jima Washington, D.C., memorial free png](https://www.pngfuel.com/free-png/opgid) for Heroic Epic and Morale promotion
[Arc de Triomphe](https://thenounproject.com/search/?q=Arc+de+Triomphe&i=789893) By Andrejs Kirma, LV for National Epic
[Bank](https://thenounproject.com/term/bank/213472/) by By P Thanga Vignesh for National Treasury
reused general icon for Great Generals and Quick Study promotions
Un-retired Mass Media icon so it can be used in mods and in unciv when it will become vanilla
* implemented uniques
"Cost increases by [30] per owned city"  - All vanilla national wonders that require something in all cities have it
"Nuclear weapon" - Game option "nuclearWeaponsEnabled" enables units with this unique
"Requires [building]" - Apollo Program, Manhattan Project etc
"Requires [policy]" - in expansions Statue of Zeus will require "Honor", Prora - "Autocracy", Landsknecht - "Mercenary Army".

### CODE REVIEW

File: core/src/com/unciv/models/ruleset/unit/BaseUnit.kt [hunk 1]
```diff
@@ -135,9 +135,11 @@ class BaseUnit : INamed, IConstruction {
         if (uniqueTo!=null && uniqueTo!=civInfo.civName) return "Unique to $uniqueTo"
         if (civInfo.gameInfo.ruleSet.units.values.any { it.uniqueTo==civInfo.civName && it.replaces==name }) return "Our unique unit replaces this"
         if (!civInfo.gameInfo.gameParameters.nuclearWeaponsEnabled
-                && uniques.contains("Requires Manhattan Project")) return "Disabled by setting"
-        if (uniques.contains("Requires Manhattan Project") && !civInfo.hasUnique("Enables nuclear weapon"))
-            return "Requires Manhattan Project"
+                && uniques.contains("Nuclear weapon")) return "Disabled by setting"
+        for (unique in uniqueObjects.filter { it.placeholderText == "Requires []" })
+            if (unique.params[0] in civInfo.gameInfo.ruleSet.buildings) {
```
Comment (yairm210) [inline core/src/com/unciv/models/ruleset/unit/BaseUnit.kt:140]: Separate unique.params[0] to val filter

File: core/src/com/unciv/logic/battle/Battle.kt [hunk 2]
```diff
@@ -217,13 +217,13 @@ object Battle {
     }
 
     private fun tryGetCultureFromKilling(civUnit:ICombatant, defeatedUnit:MapUnitCombatant){
-        //Aztecs get half the strength of the unit killed in culture and honor opener does the same thing.
-        //They stack. So you get culture equal to 100% of the dead unit's strength.
+        //Aztecs get melee strength of the unit killed in culture and honor opener does the same thing.
+        //They stack. So you get culture equal to 200% of the dead unit's strength.
         val civInfo = civUnit.getCivInfo()
         if (defeatedUnit.getCivInfo().isBarbarian() && civInfo.hasUnique("Gain Culture when you kill a barbarian unit"))
-            civInfo.policies.addCulture(max(defeatedUnit.unit.baseUnit.strength, defeatedUnit.unit.baseUnit.rangedStrength) / 2)
```
Comment (yairm210) [inline core/src/com/unciv/logic/battle/Battle.kt:224]: You sure about this?

Comment (HadeanLake) [reply core/src/com/unciv/logic/battle/Battle.kt:224]: Yes, did experiments in civ 5

**Comment by yairm210** [comment]
I don't like the national epic image, because it's a line drawing - please try to avoid those

File: core/src/com/unciv/logic/battle/Battle.kt (hunk 2 above)
Comment (yairm210) [reply core/src/com/unciv/logic/battle/Battle.kt:224]: 👍

**Comment by HadeanLake** [comment]
2020-09-21-20:41:16_1920x1080_scrot

**Comment by yairm210** [comment]
Opaaaa much better

**Pull request merged**